.github/workflows/release-node.yml

```diff
@@ -9,30 +9,41 @@ on:
       apify_version:
         description: 'Apify SDK version (e.g.: ^1.0.0)'
         required: true
-      node_version:
-        description: 'NodeJs version for image actor-node'
-        required: false
-        default: '14'
 
   repository_dispatch:
     types: [ build-images ]
 
 env:
   RELEASE_TAG: ${{ github.event.inputs.release_tag || github.event.client_payload.release_tag }}
   APIFY_VERSION: ${{ github.event.inputs.apify_version || github.event.client_payload.apify_version }}
-  NODE_VERSION: ${{ github.event.client_payload.node_version || github.event.inputs.node_version || '14' }}
+  NODE_LATEST: 16
 
 jobs:
-  # Build master images that are not dependent on existing builds.
+  # Build master images that are not dependent on existing builds.   
   build-main:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         image-name: [node]
+        node-version: [14, 15, 16]
     steps:
       -
         name: Checkout
         uses: actions/checkout@v2
+      - 
+        name: Prepare image tags
+        id: prepare-tags
+        uses: actions/github-script@v4
+        env:
+          CURRENT_NODE: ${{ matrix.node-version }}
+          LATEST_NODE: ${{ env.NODE_LATEST }}
+          RELEASE_TAG: ${{ env.RELEASE_TAG }}
+          IMAGE_NAME: apify/actor-${{ matrix.image-name }}
+        with:
+          result-encoding: string
+          script: |
+             const script = require("./.github/scripts/prepare-image-tags.js");
+             return script()
```

Comment:
might be worth renaming `script` to `generateTags`. It would make it obvious that the return value are tags without having to read the source code of the script.